bin/zerobufCxx.py

```diff
@@ -15,6 +15,8 @@
 import sys
 from collections import namedtuple, OrderedDict
 
+inline_implementation = False
```

Comment:
I think that it won't work. In Linux these functions are not inline and I would not force the use of inline functions.